src/routes/docs/tooling/keyboard/+page.markdoc

```diff
@@ -0,0 +1,81 @@
+
+## Global shortcuts {% #global-shortcuts %}
+
+Developers can utilize global shortcuts anywhere in the Appwrite Console, allowing instant access to these features, whether on their account page or creating an organization page.
+
+|PC (Windows / Linux)|macOS|Action|
+|----|----|----|
+|`Ctrl` + `K`|`⌘` + `K`|Access Command Center|
+
+
+|Shortcut|Action|
+|----|----|
+|Set theme to light|`T` then `L`|
+|Set theme to dark|`T` then `D`|
+|Set theme to dark|`T` then `A`|
+|Ask the AI|`A` then `I`|
+
+## Project shortcuts {% #project-shortcuts %}
+
+Once they create a project, developers can utilize project shortcuts. These shortcuts allow them to access the project's database, storage, messaging, auth, settings, and overview.
+
+|Shortcut|Action|
+|----|----|
+|Go to Auth|`G` then `A`|
+|Go to Storage|`G` then `S`|
+|Go to Databases|`G` then `D`|
+|Go to Functions|`G` then `F`|
+|Go to Projects|`G` then `P`|
+|Go to Overview|`G` then `O`|
+|Go to Messaging|`G` then `M`|
+|Go to Settings|`G` then `E`|
+|Create Organization|`C` then `O`|
+
+# The Appwriter {% #the-appwriter %}
+
+{% only_dark %}
+![The Appwriter](/images/docs/keyboard/dark/appwriter.png)
+{% /only_dark %}
+
+{% only_light %}
+![The Appwriter](/images/docs/keyboard/appwriter.png)
+{% /only_light %}
+
+The Appwriter is an exclusive keyboard custom-designed by the Appwrite team. It is optimized to benefit developer productivity and is specially tuned to use with the Appwrite Console.
+
+The Appwriter assigns Appwrite icons from the Appwrite Console and Docs to specific keys, making memorizing keyboard shortcuts easier. For example, `G` then `D` is the shortcut for going to databases, and the icon on the keyboard matches the icon in Appwrite Databases.
+
+## Specifications
+
+The box includes the Appwriter keyboard, a USB-C data/charging cable, a 2-in-1 keycap and switch puller, extra PC layout keycaps, and an information card with a QR code. The keyboard comes with Mac layout keys pre-installed for MacOS or PC specific keys.
+
+||Spec|
+|----|----|
+|Battery|3000mAh Rechargeable Battery|
+|Design|75% Compact Design Keyboard|
+|Keycaps|84 Custom Dye Sublimated PBT Keycaps|
+|Switches|Gateron G Pro Yellow Pre-Lubed Switches|
+|Connections|Tri-Mode Connectivity (2.4Ghz Wireless / Bluetooth / Wired USB-C)|
+|Case|Noise dampening case-foam|
+|RGB Backlight|16 million colours & multiple lighting effects|
+|Switch Support|Hot-Swappable switches with 3 & 5 pin|
+|N-Key|N-Key rollover|
```

Comment:
We need to stick to sentence casing here unless we are referring to a proper noun